users/templates/base.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
please fix the indentation
```suggestion
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1">
```

Reply:
done.